### PR TITLE
Automatically update license info in Pull Requests

### PR DESCRIPTION
**Pull request by rreich**

### CODE REVIEW

File: studio-server/pom.xml [hunk 1]
```diff
@@ -17,6 +17,13 @@
   </properties>
 
   <repositories>
+    <repository>
+      <id>look-here-first</id>
+      <url>https://repo1.maven.org/maven2/</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
```
Comment (ahubold) [inline studio-server/pom.xml:26]: Why is this necessary for this PR? (same for pluginRepositories below)
Is it just for faster builds, or is it required for the GitHub action?
If needed, maybe add a comment here?

**Review by ahubold — COMMENTED** [review]
Very cool solution! :+1: 

only one question...

File: studio-server/pom.xml (hunk 1 above)
Comment (rreich) [reply studio-server/pom.xml:28]: It's just optimization. I added a comment: [9925a9a](https://github.com/CoreMedia/content-hub-adapter-rss/pull/6/commits/9925a9ac1ab550c925b2e631412b5e60cf7a088f)

**Review by ahubold — APPROVED** [review]
